docs/usage/docker.md

```diff
@@ -206,6 +206,37 @@ module.exports = {
 };
 ```
 
+#### Google Container Registry
+
+Assume you are running GitLab CI in the Google Cloud and are storing the Docker images in the Google Container Registry (GCR).
+Access to the GCR uses a Bearer token based authentication.
+The token for the build can be obtained by running `gcloud auth print-access-token`, which requires the Google Cloud SDK to be installed.
+
+It is also very important to note that this is a short-lived token ([60 minutes](https://stackoverflow.com/questions/50370714/google-cloud-bearer-token-expiry)) and thus storing it repeated builds is useless.
+
+When applying this to Renovate this all means the access token must be injected into the `hostRules` configuration just before Renovate is started.
+
+_This documentation only gives **a few hints** on **a possible way** to achieve this in this scenario_
+
+To get access to the token a custom Renovate Docker image is needed that includes the Google Cloud SDK.
+The Dockerfile can look like this:
+
+```Dockerfile
+FROM renovate/renovate:25.40.1
```

Comment:
Note: we need to add this to our docs renovate upgrade. So this version gets updated regularly.

Reply:
Oh, I thought it would get picked up automatically???

Reply:
Not sure, maybe yes. We should check it after merge.

Reply:
I think it matches all `*.md` files, at least if I'm reading `matchPaths` correct?

```json
    {
      "description": "Update references to Docker tags for renovate/renovate in Markdown files weekly",
      "matchPackageNames": ["renovate/renovate"],
      "matchPaths": [".md"],
      "extends": ["schedule:weekly"],
      "automerge": true,
      "stabilityDays": 0,
      "commitMessageTopic": "references to {{{depName}}}",
      "semanticCommitType": "docs",
      "semanticCommitScope": null,
      "additionalBranchPrefix": "docs-"
    },
```

https://github.com/renovatebot/.github/blob/930726ea9db00b581216bc7618a832dfcd81155a/default.json#L44-L55

Reply:
Sure, does the regex manager match too?

Reply:
```json
  "regexManagers": [
    {
      "description": "Update Renovate references in Markdown files",
      "fileMatch": ["\\.md$"],
      "matchStrings": [
        "(?<depName>renovate\\/renovate):(?<currentValue>[a-z0-9.-]+)(?:@(?<currentDigest>sha256:[a-f0-9]+))?"
      ],
      "datasourceTemplate": "docker",
      "versioningTemplate": "docker"
    },
```

https://github.com/renovatebot/.github/blob/930726ea9db00b581216bc7618a832dfcd81155a/default.json#L96-L105

Reply:
Ok, should work as is. 👍

Reply:
Ok, so I'll leave this as-is.